tests/conftest.py

```diff
@@ -17,25 +20,39 @@ def logger():
 
 
 @fixture()
-def mocked_item(mocked_session):
+def mocked_item(mocked_session, mocked_module):
     """Mock Pytest item for testing."""
     test_item = mock.Mock()
     test_item.session = mocked_session
+    test_item.fspath = py.path.local('/path/to/test')
+    test_item.name = 'test_item'
+    test_item.parent = mocked_module
     return test_item
 
 
+@fixture()
+def mocked_module(mocked_session):
+    """Mock Pytest Module for testing."""
+    mocked_module = mock.Mock()
+    mocked_module.parent = mocked_session
+    return mocked_module
+
+
 @fixture()
 def mocked_config():
     """Mock Pytest config for testing."""
     mocked_config = mock.create_autospec(Config)
     mocked_config._reportportal_configured = True
+    mocked_config.rootdir = py.path.local('/path/to')
+    mocked_config.trace = TagTracer().get('root')
+    mocked_config.pluginmanager = mock.Mock()
     return mocked_config
 
 
 @fixture()
 def mocked_session(mocked_config):
     """Mock Pytest session for testing."""
-    mocked_session = mock.Mock()
+    mocked_session = Session(mocked_config)
```

Comment:
That's not a good idea to use pytest real class instead of a mock because it may do something that you do not see under the hood.

Reply:
Is it OK to write like this: `mock.Mock(Session)`? I need to bypass `isinstance` test, so that's why I'm stick to the class.

Reply:
What is the isinstance test? I do not see this function call in our tests and agent code. I suggest using create_autospec if you want to create a mock object using another object.